packages/core/src/App/Components/Elements/Modals/RedirectNotice/redirect-notice-modal.jsx

```diff
@@ -32,10 +33,14 @@ const RedirectNoticeModal = ({ is_logged_in, is_eu }) => {
     };
 
     const onClickExternalLink = e => {
-        if (isThirdPartyLink(e.target.href) && is_logged_in && is_eu) {
-            setExternalLink(e.target.href);
-            e.preventDefault();
-            setDialogStatus(true);
+        const linkElement = e.target.tagName === 'A' ? e.target : e.target.closest('a');
+        if (linkElement) {
```

Comment:
```suggestion
        if (!!link_element) {
```

Reply:
This will make sure the element is boolean value

Reply:
@akmal-binary  It's failing the test though, giving a `Redundant double negation` error. Any way around it?

Reply:
Hmm, you can remove it then, probably test knows when to use it and when to avoid it since it doesn't fail on other use cases